src/modifiers/distribute_along_even.gd

```diff
@@ -3,9 +3,12 @@ extends "base_modifier.gd"
 
 
 export(float) var interval = 1.0
-export(float) var offset = 0.0
+export(float, 0, 100) var offset = 0.0
```

Comment:
Unless I got the math wrong, isn't offset in unit values instead of being a percent? In that case this variable shouldn't be locked to a specific range.

Reply:
It is in unit values, the bounds are unnecessary, I remember seeing an error with the offset being between -1, 1 mentioned, and I was playing around with it but forgot to remove, thats what most of these things are lol